src/libraries/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -58,7 +59,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Arg_EmptyOrNullString" xml:space="preserve">
```

Comment:
I recommend renaming this, since it no longer matches the string of the same name in other libraries (and it'll help make sure all callers are passing the argument).